### ISSUE TITLE
🐛 [css] false positive on active-view-transition-type with multiple names

### ISSUE THREAD

**Issue by ivany4** [biomejs/biome#8472]
### Environment information

```block
CLI:
  Version:                      2.3.9
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v24.11.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/10.9.3

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               false
  VCS enabled:                  false

Workspace:
  Open Documents:               0
```

### What happened?

When using `html:active-view-transition-type(forwards, backwards) {...}` from the [MDN example](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Selectors/:active-view-transition-type) (also syntax mentioned in the [spec](https://www.w3.org/TR/css-view-transitions-2/#the-active-view-transition-type-pseudo)), the following error is displayed: `Expected an identifier but instead found ','.`

[Repro in Playground](https://biomejs.dev/playground/?code=aAB0AG0AbAA6AGEAYwB0AGkAdgBlAC0AdgBpAGUAdwAtAHQAcgBhAG4AcwBpAHQAaQBvAG4ALQB0AHkAcABlACgAZgBvAHIAdwBhAHIAZABzACwAIABiAGEAYwBrAHcAYQByAGQAcwApACAAewAKACAAIAAvACoAIAAuAC4ALgAgACoALwAKAH0A&language=css)

### Expected result

No error

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
